theme/assets/css/src/components/top-app-bar.css

```diff
@@ -145,4 +145,13 @@
 	& .mdc-drawer__close {
 		display: none;
 	}
+
+	& .dark-mode__button {
```

Comment:
Is there a fallback?

Reply:
@rodydavis This turns into `.top-app-bar .dark-mode__button` in the compiled file